.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: julia
 julia:
-    - 0.4
```

Comment:
Yes please, otherwise JuMP (and anyone who has JuMP installed) will be stuck on ForwardDiff 0.1 for a long time